### PR TITLE
Fix alternative generation when original ratio is extreme

### PR DESCRIPTION
**Pull request by ngilain**
[OO-BUGFIX] Fix alternative generation when original image ratio is extreme

### CODE REVIEW

File: MediaAdmin/FileUtils/Image/ImagickImageManager.php [hunk 1]
```diff
@@ -11,13 +11,19 @@
...
+    protected $maxWidth;
+    protected $maxHeight;
 
     /**
      * @param ImagickFactory $imagickFactory
+     * @param int            $maxWidth
+     * @param int            $maxHeight
      */
-    public function __construct(ImagickFactory $imagickFactory)
+    public function __construct(ImagickFactory $imagickFactory, $maxWidth = 5000, $maxHeight = 5000)
```
Comment (veolys-nanne) [inline MediaAdmin/FileUtils/Image/ImagickImageManager.php]: I don't understand why you add default values as you inject them in service definition.

Comment (ngilain) [inline MediaAdmin/FileUtils/Image/ImagickImageManager.php]: It's useless, you're right, I remove this

**Comment by ngilain** [comment]
@itkg-nanne : pr updated

**Comment by veolys-nanne** [comment]
:+1:

**Pull request merged**